lib/InputWithButton/index.js

```diff
@@ -1,8 +1,32 @@
 import React, { Component } from 'react';
-import cx from 'classnames';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
+import Input from '../core/Input';
 import Button from '../Button';
 
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const LeftHandInput = styled(Input)`
+  width: 220px;
+  border-radius: 4px 0px 0px 4px;
+`;
+
+const RightHandButton = styled(Button)`
```

Comment:
I was kind of hoping to see some deleted styles, to get an idea of the refactors from `scss` to this. Could we do that?

Reply:
We have one deleted `scss` file here https://github.com/gathercontent/gather-ui/pull/533/files#diff-07096a4f84efc8ff7ff11c7cf5b8b024